File: FlightBookingSystem/ClientApp/src/app/api/models/booking-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from '../models/time-place-rm';
export interface BookingRm {
  airline?: string | null;
  arrival?: TimePlaceRm;
  departure?: TimePlaceRm;
  flightId?: string;
  numberOfBookedSeats?: number;
  passengerEmail?: string | null;
  price?: string | null;
}

File: FlightBookingSystem/ReadModels/BookingRM.cs
namespace FlightBookingSystem.ReadModels
{
    public record BookingRM(
        Guid FlightId, 
        string Airline, 
        string Price, 
        TimePlaceRm Departure, 
        TimePlaceRm Arrival, 
        int NumberOfBookedSeats, 
        string PassengerEmail);
}
